### PR TITLE
Update README with notes on OS compatibility and required package installation

### PR DESCRIPTION
**Pull request by teschmitt**
Adds notes on OS compatibility and required package installation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ The current features are:
 - (with extendability for new convergence layer adapters, routing algorithms, and storage managers)
 
 ## Getting Started
+
+Installation has been successfully tested on Windows and Linux (x86_64 and arm64). The required [mpy-cross](https://pypi.org/project/mpy-cross/) has been shown to have installation issues on Mac M1s under Mac OS X and may have to be built and installed manually.
+
```
Comment (lh70) [inline README.md:18]: This is valuable information, but it may be better suited at the beginning of the "MicroPython Installation Guide" section.

**Review by lh70 — CHANGES_REQUESTED** [review]
Thank you very much for testing and improving the Installation on the ESP32 with Mac and Linux.
There is only a single paragraph I would move to a different section. Please take a look at the details below.

File: README.md (hunk 1 above)
Comment (teschmitt) [reply README.md:18]: I've moved the paragraph to the beginning of the MicroPython section and additionally added placeholders in the `wlan.json` to make wifi configuration clearer.

Comment (lh70) [reply README.md:18]: Thank you for the contribution. The placeholders for clarity are also a good idea.

**Review by lh70 — APPROVED** [review]

**Pull request merged**